### ISSUE TITLE
Not working following instructions

### ISSUE THREAD

**Issue by Syavne** [tentwentyfour/remoteauth-ng#2]
Good day.

After following configuration instructions, even when REMOTE_USER is present (tested with a phpInfo function) MediaWiki 1.28 still says "Not logged in".

No errors in log files from Apache 2.4.6-45.el7.centos and php 5.5.38-1.w7, we are using a clean install on a fully upgraded CentOS 7 virtual machine.

Tried noris-network/mediawiki-extensions-sessionprovider-remoteuser with the same results.

Is there a particular place where we can look for debug messages or errors?.

**Comment by dattn** [comment]
Do have you created the user ?
The user needs to be created manually.

In the latest version (master) there is an option to automatically create a user if not exists.

**Comment by Syavne** [comment]
Yes, every user exist and I'm using master. With some modifications to the code I can say for sure that REMOTE_USER is there and the extension is indeed reading it, however, I'm having problems trying to debug the Mediawiki session as I don't know yet how it works (lack of time), I've been playing with different cache settings also.
